gravitee-apim-console-webui/src/services/group.service.ts

```diff
@@ -13,9 +13,28 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import * as _ from 'lodash';
 import { PageQuery } from '../entities/pageQuery';
 
+function mapRoles(roles = []) {
+  return Object.entries(roles)
+    .filter(([scope]) => !!scope)
+    .map(([scope, name]) => {
+      return { scope, name };
+    });
+}
+
+function mapMembers(members = []) {
```

Comment:
since there is no `export` maybe it can just be private method of the `GroupService`